### PR TITLE
Wrap gnome control center and online accounts

### PR DESCRIPTION
**Pull request by selaux**
###### Motivation for this change

I could not add an exchange account using `gnome-online-accounts` because it could not find `glib_networking`

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/desktops/gnome-3/3.22/core/gnome-control-center/default.nix [hunk 1]
```diff
@@ -37,15 +37,6 @@ stdenv.mkDerivation rec {
     substituteInPlace panels/datetime/test-endianess.c --replace "/usr/share/locale/" "$out/share/locale/"
   '';
 
-  preFixup = with gnome3; ''
-    wrapProgram $out/bin/gnome-control-center \
-      --set GDK_PIXBUF_MODULE_FILE "$GDK_PIXBUF_MODULE_FILE" \
-      --prefix XDG_DATA_DIRS : "${gnome3.gnome_themes_standard}/share:${sound-theme-freedesktop}/share:$out/share:$out/share/gnome-control-center:$XDG_ICON_DIRS:$GSETTINGS_SCHEMAS_PATH"
-    for i in $out/share/applications/*; do
-      substituteInPlace $i --replace "gnome-control-center" "$out/bin/gnome-control-center"
```
Comment (bjornfor) [inline pkgs/desktops/gnome-3/3.22/core/gnome-control-center/default.nix]: Did you check that this substituteInPlace line is not needed anymore?

Comment (selaux) [inline pkgs/desktops/gnome-3/3.22/core/gnome-control-center/default.nix]: Thanks for spotting, that was still needed. I added it back.

**Comment by selaux** [comment]
Any chance for a merge?

**Review by 0xABAB — APPROVED** [review]

**Comment by eqyiel** [comment]
Thanks!

**Comment by selaux** [comment]
Rebased.

**Pull request merged**